redis/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM redis:latest
+
+COPY redis.conf /usr/local/etc/redis/redis.conf
+
+CMD [ "redis-server", "/usr/local/etc/redis/redis.conf" ]
+
+RUN usermod -u 1000 www-data
```

Comment:
I don't think it's needed for a redis container. Also, could you add a newline at the end of file?